style.css

```diff
@@ -0,0 +1,102 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;1,500&display=swap');
+@import url("color-pallete/color.css");
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
+    background-color:var(--primary-color);
+    word-wrap: break-word;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    color: var(--font-color);
+}
+
+.nav {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    padding: 0.2rem 2rem;
+}
+
+.nav-center {
+    display: flex;
+    justify-content: end;
+    align-items: flex-end;
+    flex-grow: 1;
+    gap: 2rem;
+    font-size: 1.25rem;
+    cursor: pointer;
+}
+
+.nav-center h4:hover{
+    color:#764abc;
+}
+.logo-container {
+    display: flex;
+    flex-direction: row;
+    padding: 10px;
+    cursor: pointer;
+}
+
+.logo-container h2{
+ color: #764abc
+}
+
+
+main{
```

Comment:
Avoid general selector as much as possible . Because is you want to use it in any website we don't want it to hinder with anything else until and unless its resetting or very general setting. I hope you got my point you class instead

Reply:
Sure.I'll keep this in mind from now on